website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -10,10 +10,91 @@ These components integrate SageMaker with the portability and orchestration of K
 
-## Tutorials
+Currently, we support two versions of these SageMaker components - the traditional v1 components and the ACK Based v2 components. The Sample included with Kubeflow is currently based off of the v2 Training Component. However, you can just as easily configure and use all others by referring to our more detailed guides over at the pipelines repo - 
+1. [AWS Samples for SageMaker Kubeflow Pipeline Components](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples)
+2. We have also added a new sample which allows you to run the v1 and v2 components in the same pipeline. Refer to this [README on specific steps](https://github.com/kubeflow/pipelines/tree/73d45eeb7136bb23f9d7e103f50a226f111c3143/components/aws/sagemaker/TrainingJob/samples/mnist-kmeans-classification)  # TODO: this links to a commit
+
+## Tutorial: SageMaker Training Pipeline for MNIST Classification with K-Means
 
 Kubeflow on AWS includes pipeline tutorials for SageMaker components. Examples are included for the most common features that can be used to run production loads with just a few clicks. To try out the examples, deploy Kubeflow on AWS on your cluster and visit the Kubeflow Dashboard `Pipelines` tab.
 
-To train a classficiation model using SageMaker Components for Kubeflow Pipelines, see the [MNIST classification with KMeans](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/mnist-kmeans-sagemaker) sample. 
\ No newline at end of file
+In the following section we will walk through the steps to run the Sample SageMaker Training Pipeline. This sample runs a pipeline to train a classficiation model using Kmeans with MNIST dataset on SageMaker. This example was taken from an existing [SageMaker example](https://github.com/aws/amazon-sagemaker-examples/blob/8279abfcc78bad091608a4a7135e50a0bd0ec8bb/sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb) and modified to work with the Amazon SageMaker Components for Kubeflow Pipelines. 
+
+## Installing Kubeflow Pipelines
+There are two ways to deploy Kubeflow Pipelines on AWS. If you are using the broader Kubeflow deployment, ACK and KFP come configured and ready to use with any of the Kubeflow on AWS deployment options. Read on to complete the pipeline execution. 
+
+On the other hand, if you are using standalone KFP installation, please refer to our more detailed instructions over at the [AWS Docs for Kubeflow Pipeline Components]().
+
+## S3 Bucket
+To train a model with SageMaker, we need an S3 bucket to store the dataset and artifacts from the training process. Run the following commands to create an S3 bucket. Specify the value for `SAGEMAKER_REGION` as the region you want to create your SageMaker resources. For ease of use in the samples (using the default values of the pipeline), we suggest using `us-east-1` as the region.
+
+```
+export SAGEMAKER_REGION=us-east-1
+export S3_BUCKET_NAME="data-bucket-${SAGEMAKER_REGION}-$RANDOM"
```

Comment:
$RANDOM is not working on some os, lets remove it and ask user to specify a name?

Need to remove from official sample as well later